data/generateData.js

```diff
@@ -177,8 +181,11 @@ function getTopRoundsByScore(roundsByScore) {
   return topRounds;
 }
 
-function getStats(rounds) {
-  const statsByPlayer = Object.values(rounds.reduce(getStatsByPlayer, {}));
+function getStats(rounds, maxRoundsPerPlayer) {
+  const boundStatsByPlayer = getStatsByPlayer.bind(null, maxRoundsPerPlayer);
+  // Recent rounds first so the maxRoundsPerPlayer will the right ones
+  rounds.sort((a, b) => new Date(b.date) - new Date(a.date));
+  const statsByPlayer = Object.values(rounds.reduce(boundStatsByPlayer, {}));
```

Comment:
This `reduce` stuff still confuses me, but it's certainly in-line with what the rest of this file does, so if it works, I'm fine with it.

Reply:
I'm also not a fan of it. I'm trying to make a minor tweak, not do heart surgery.